estimation/model.py

```diff
@@ -225,7 +225,7 @@ def save_model(self, output_file: str):
 
         self._model.save(output_file)
 
-        with h5py.File(output_file, "r+") as f:
+        with h5py.File(output_file, "r") as f:
```

Comment:
Is this a merge issue?

Reply:
This change is incorrect, this is the save function so it requires write access. So this change should be reverted.